components/section/info/src/index.js

```diff
@@ -9,7 +9,7 @@ class SectionInfo extends Component {
 
     return (
       <section className={baseClass}>
-        <h3 className={`${baseClass}-title`}>{title}</h3>
+        { title !== undefined && <h3 className={`${baseClass}-title`}>{title}</h3> }
```

Comment:
what if title is empty string? maybe you should use !title

Reply:
In the description of the component it says: `In the In addition you can provide an empty string if the Title space is needed.`.

Reply:
Manage spacing with empty strings looks a bit weird, isn't it?